pkgs/os-specific/linux/lxc/default.nix

```diff
@@ -41,10 +40,10 @@ stdenv.mkDerivation rec {
     "--disable-api-docs"
     "--with-init-script=none"
     "--with-distro=nixos" # just to be sure it is "unknown"
-  ] ++ optional (libapparmor != null) "--enable-apparmor"
-    ++ optional (libselinux != null) "--enable-selinux"
-    ++ optional (libseccomp != null) "--enable-seccomp"
-    ++ optional (libcap != null) "--enable-capabilities"
+  ] ++ lib.optional (libapparmor != null) "--enable-apparmor"
```

Comment:
Shouldn't this be flags like withAppArmor?

Reply:
yes, but I'm not sure how to change it without breaking other people setups

Reply:
We could default it to true.

Reply:
yes, I think it should be fine, but I'd prefer changing it in separate PR